packages/react/src/components/PageTitleBar/_page-title-bar.scss

```diff
@@ -152,7 +152,7 @@
     flex-direction: row;
     align-items: center;
     padding: $spacing-04 0 $spacing-02 $spacing-07;
-    max-height: clamp(0rem, $spacing-02, $spacing-08);
+    max-height: clamp(0rem, #{$spacing-07}, #{$spacing-08});
```

Comment:
Yeah, I don't know why it was ever added as clamp to begin with, but I assumed it had a reason and left it in place for some edge case rather than remove it. 🤷‍♂️

Reply:
removed the clamp. It seems to be fine. Can you give it another look after netlify deploys?

Reply:
Looks good